nntrainer/tensor/cl_operations/blas_kernel_strings.cpp

```diff
@@ -717,33 +717,94 @@ const std::string &getConcatClAxis1Kernel() {
 
 const std::string &getRMSNormClKernel() {
   static const std::string rmsnorm_cl_kernel_ =
-    R"(__kernel void rmsnorm_cl(
+    R"(
+  #pragma OPENCL EXTENSION cl_intel_subgroups : enable  
+  #pragma OPENCL EXTENSION cl_intel_required_subgroup_size : enable
+
+ __attribute__((intel_reqd_sub_group_size(32)))
```

Comment:
This feature is only compatible with Intel GPUs. Let's add a condition for non-Intel GPUs.

```suggestion
    #ifdef cl_intel_required_subgroup_size
    #pragma OPENCL EXTENSION cl_intel_subgroups : enable
    #pragma OPENCL EXTENSION cl_intel_required_subgroup_size : enable
    #define INTEL_GPU 1
    #define REQD_SUBGROUP_SIZE_32 __attribute__((intel_reqd_sub_group_size(32)))
    #elif defined(cl_qcom_reqd_sub_group_size)
    #pragma OPENCL EXTENSION cl_qcom_reqd_sub_group_size : enable
    #define ADRENO_GPU 1
    #define REQD_SUBGROUP_SIZE_64 __attribute__((qcom_reqd_sub_group_size("half")))
    #endif
    
    #ifdef INTEL_GPU
    REQD_SUBGROUP_SIZE_32
    #elif defined(ADRENO_GPU)
    REQD_SUBGROUP_SIZE_64
    #endif
```

Reply:
OK, good idea. I'll do it, thanks :)